appveyor-nightly.yml

```diff
@@ -27,7 +27,8 @@ environment:
   KEEP_VIGNETTES: true
   R_ARCH: x64
   R_VERSION: 4.1.0
-  R_CHECK_ARGS: --no-multiarch --no-manual --as-cran
+  R_CHECK_ARGS: --no-multiarch --no-manual --as-cran --no-build-vignettes
```

Comment:
I don't see any gain in this step. Vignettes are just built and then are tested again in the installed package.

The only use case I see where this could fail: an example file, which is used in the vignettes, is explicitly excluded from the package via **.RBuildIgnore**. 

In all other cases: if vignette **creation** was successful - vignette **check** will be successful too.